L1Trigger/L1TMuonEndCap/plugins/L1TMuonEndCapTrackProducer.cc

```diff
@@ -41,6 +41,7 @@ using namespace L1TMuon;
 L1TMuonEndCapTrackProducer::L1TMuonEndCapTrackProducer(const PSet& p) {
 
   inputTokenCSC = consumes<CSCCorrelatedLCTDigiCollection>(p.getParameter<edm::InputTag>("CSCInput"));
+  bxShiftCSC = p.getUntrackedParameter<int>("CSCInputBxShift", 0);
```

Comment:
Hi Andrea,

Thanks for spotting this.  I had tried "getParameter" instead of
"getUntrackedParameter" and it segfaulted.  I'm not familiar with the rules
and conventions for PSet parameters - someone will either have to educate
me, or anyone who knows what they're doing is welcome to edit this directly.

Best,
Andrew

On Fri, May 20, 2016 at 3:55 PM, Andrea Bocci <notifications@github.com>
wrote:

> In L1Trigger/L1TMuonEndCap/plugins/L1TMuonEndCapTrackProducer.cc
> <https://github.com/cms-l1t-offline/cmssw/pull/324#discussion_r64044708>:
>
> > @@ -41,6 +41,7 @@ using namespace L1TMuon;
> >  L1TMuonEndCapTrackProducer::L1TMuonEndCapTrackProducer(const PSet& p) {
> >
> >    inputTokenCSC = consumes<CSCCorrelatedLCTDigiCollection>(p.getParameter<edm::InputTag>("CSCInput"));
> > +  bxShiftCSC = p.getUntrackedParameter<int>("CSCInputBxShift", 0);
>
> no untracked parameters, please
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-l1t-offline/cmssw/pull/324/files/7d1bef0a8fad59b1865ae5773c225e53bec47cf5#r64044708>
>